Add 2023-08-09 agenda and minutes (#552)
solid/specification: meetings/2023-08-09.md

* SC: Yes, for now just one meeting. We don't really have panel meetings happening anymore for the most part, so we just raise interesting topics to solve by those interested.
* SC: Let's continue in chat. The recurring meeting is Tuesdays at 14pm UTC. We can see who can make that, and maybe dip shortly into it at the end of this meeting.
* eP: Would it be possible to first [raise an issue on SolidOIDC repo](https://github.com/solid/solid-oidc/issues), to organise this there?
* SC: I think Ross preferred not to do that at the time.
* RH: If anyone can maybe start it, and I can comment, that would be great. I can put it there. 

### Summary of Special Topic: Alternative solutions to container HATEOAS

* SC: We had such a meeting on 08-08
* eP: We explored the topic from different angles. No information was ended, but we explored how to tackle it. The main outcome was that we dove again into HTML, so I also want to incorporate that (i.e. to have an HTML representation of a container). Personally, I thought we would just need RDF description, and the problem was client- vs server-asserted statements, but the topic was opened further.

> * WT: Main conclusion IMO was: we started exploring how to solving this question of how to combine server-managed RDF source like a container with both client-managed RDF or client-managed non-RDF representations of the same resource. We had started on a decision tree that we answered affirmative/negative way to see where it leads us.

Reviewer: ```suggestion
* WT: Main conclusion IMO was: we started exploring how to solve this question of how to combine server-managed RDF source like a container with both client-managed RDF and client-managed non-RDF representations of the same resource. We had started on a decision tree that we answered affirmative/negative ways to see where it leads us.
```

* SC: There are two change suggestions (let's call them the "short" and "long" version):
  * Short: https://github.com/solid/process/pull/323#discussion_r1282179722
  * Long: https://github.com/solid/process/pull/323#discussion_r1285660875
* SC: The short version is closest to 2023-08-02 meeting resolution, and the longer version incorporates finer details of W3C requirements as well as recommendations for the ecosystem. We can commit one of these change suggestions now, and keep the PR open as previously agreed, integrating updates as we go until the end of the month.
* WT: To clarify: as you said, it is not that the versions differ but they both say the W3C requirements and the context in which we've been working... like the deliverables that have to comply in the WG.

PROPOSAL: Commit the short version? (If long version is preferred, say so along with +0)

* eP: +0
* WT: +0
* SC: +1 (to be on the "safe side" for now.)

PROPOSAL: Commit the long version?

* eP: +1
* WT: +1
* LD: +1

> * eP: I see the long version safer. I think we often seem to disagree due to taking about different things to license. Having it detailed can help with clarity on what specifically is being discussed. 

Reviewer: ```suggestion
* eP: I see the long version safer. I think we often seem to disagree due to talking about different things to license. Having it detailed can help with clarity on what specifically is being discussed. 
```
